### DISCUSSION TITLE
TypeError: 'set' on proxy: trap returned falsish for property when trying to add data from axios request to state

### DISCUSSION THREAD

**Discussion by RSurya99** [vuejs/pinia#1048] [Help and Questions] [answered]
### Reproduction
Error when trying to add data from Axios request to state

### Expected behavior

data from request will be saved into state

### Actual behavior

error when saving the data to state

my code to save the request result to state
![piniaError2](https://user-images.githubusercontent.com/73375663/153622968-65d8596e-6b47-452d-a68a-1d9cc62b92b8.png)

error message that I get
![piniaError](https://user-images.githubusercontent.com/73375663/153623016-fb6aa70e-03af-4efd-97ab-9bec5525d299.png)

### Additional information

Vue ver 3.2.29
Pinia ver 2.0.11
Typescript ver 4.5.5

**Comment by alanhumphrey** [comment, marked as answer]
I ran into the same problem. The issue is that you have a getter and a state property with the same name. I changed my getter to a different name and the error message went away.

**Reply by RSurya99** [reply]
Thanks, it worked

**Reply by realtebo** [reply]
I love you, thanks !

i think compiler / linter should warn about getter with same name of state
